metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1262,6 +1262,21 @@ class MetalsLanguageServer(
           workspaceSymbols.indexClasspath()
         }
 
+//        val source = scala.io.Source.fromFile(bloopGlobalJsonPath.toUri)
+//        val bloobGlobalJsonContent = try source.mkString finally source.close()
+//        bloobGlobalJsonContent.parseJson.getAsJsonObject
+        if (userConfig.bloopJvmProperties != old.bloopJvmProperties) { // TODO is the comparison meaningful
+          val bloopGlobalJsonPath: Path = Paths
```

Comment:
So the logic for this can probably actually go somewhere else. Like we can check if the `bloopJvmProperties` are changed and then if so, maybe call into `BloopServers` or something and have a method in the companion object there do this sort of stuff.

Reply:
thank you so much for the input…
So I would be adding the code for bloop restart as you mentioned
Now for displaying the parameters of the bloop global json file to the user in the settings menu, we would have to load it on client side right?

Reply:
@dos65 Hey Vadim, can you please provide a tad bit of input on whether this should be solved on the client side, and if so how?

Reply:
`def shutdownServer()` and `def newServer()` are now in the class of `BloopServers`, do we want them in the companion object, or do we want to find some way to access an existing created instance of `BloopServers` and call the methods on the instance?

Reply:
Because it seems that we already have access to an instance of `BloopServers` though the `bloopServers` variable of `MetalsLanguageServers`
So I guess I would go the direction of calling them on the instance…

Reply:
We have the method `ensureDesiredVersion` in `BloopServers` which I think should be blended with some `ensureDesiredJvmSettings` functionality to achieve the purpose...